proto/exocore/appchain/common/v1/wire.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+
+package exocore.appchain.common.v1;
+
+import "cosmos/staking/v1beta1/staking.proto";
+import "gogoproto/gogo.proto";
+import "tendermint/abci/types.proto";
+
+option go_package = "github.com/ExocoreNetwork/exocore/x/appchain/common/types";
+
+// This file containts all of the information that is sent over the wire by either
+// the coordinator or each of the subcribers.
+
+message HandshakeMetadata {
+  // coordinator_fee_pool_addr is the address on the coordinator to which the 
+  // subscriber chain will send the fees proportionally and periodically.
+  string coordinator_fee_pool_addr = 1;
+  // version is the version of the appchain protocol
+  string version = 2;
```

Comment:
Though we have the version, Is it necessary to add a field to verify the compatibility between the coordinator and the subscriber?